src/system_data.jl

```diff
@@ -144,10 +154,12 @@ function add_time_series_from_file_metadata!(
     file_metadata::Vector{TimeSeriesFileMetadata};
     resolution = nothing,
 ) where {T <: InfrastructureSystemsComponent}
-    cache = TimeSeriesParsingCache()
-    for metadata in file_metadata
-        if resolution === nothing || metadata.resolution == resolution
-            add_time_series_from_file_metadata_internal!(data, T, cache, metadata)
+    open_time_series_store!(data, "r+") do
```

Comment:
This will speedup time series additions from our JSON/CSV files, which is used by the table data parser.